RELEASE-NOTES.rst

```diff
@@ -1,54 +1,15 @@
 ===============================
- Flask-IIIF v0.2.0 is released
+ Flask-IIIF v0.2.1 is released
 ===============================
 
-Flask-IIIF v0.2.0 was released on May 22, 2015.
+Flask-IIIF v0.2.1 was released on May 22, 2015.
 
 About
 -----
 
 Flask-IIIF is a Flask extension permitting easy integration with the
 International Image Interoperability Framework (IIIF) API standards.
 
-Incompatible changes
---------------------
-
-- Removes `uuid_to_path_handler` callback.
-
-- Updates error classes names (MultimediaImageResizeError and
-  MultimediaImageCropError).
-
-New features
```

Comment:
New features
------------

- Adds new signals before and after image info requests.